assets/sidecars/configmap_provisioner_role.yaml

```diff
@@ -0,0 +1,8 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: configmap-external-provisioner-role
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list"]
```

Comment:
https://github.com/openshift/ibm-vpc-block-csi-driver-operator/blob/master/assets/rbac/provisioner_role.yaml#L87-L90

Reply:
@dobsonj You were involved in https://github.com/openshift/ibm-vpc-block-csi-driver-operator/pull/40, and [this comment](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/pull/40/commits/ff4825c94286a4bb9cdcc3d74442fb0e0b3fd6f1#r901363637):

> adding the resources and permissions for provisioner role should suffice. It can be removed from resizer and registrar.

suggests that "configmaps" rule was added intentionally. Do you remember why `external-provisioner` needs it and what makes `ibm-vpc-block-csi-driver-operator` so special that other operators are happy w/o it?

Reply:
AFAIK, [this configmap](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/blob/master/assets/configmap.yaml) is needed by the CSI driver container, but I can't think of a reason why external-provisioner would need it.